<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="scaleConstEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="shiftBtn">
       <property name="text">
        <string>Shift</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QComboBox" name="onemodelBox"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="addConstEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="13" column="3">
      <widget class="QPushButton" name="mappingBtn">
       <property name="text">
        <string>Make mapping</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/gui.qrc">
         <normaloff>:/map.png</normaloff>:/map.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Operations on multiple datasets</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QPushButton" name="globalfitBtn">
       <property name="text">
        <string>Global fit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="monscaleEdit">
       <property name="text">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="rebinBtn">
       <property name="text">
        <string>Rebin</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add a constant:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="mulBtn">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="shiftConstEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="addBtn">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Scale with constant:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Use one model for all:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Shift X values:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Change monitor:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="monscaleBtn">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="onemodelBtn">
       <property name="text">
        <string>Do it</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Rebin individual sets:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QPushButton" name="paramsetBtn">
       <property name="text">
        <string>Do it</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Create dataset from parameter(s):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="precisionEdit">
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QLineEdit" name="mergeEdit">
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="floatMergeBtn">
         <property name="text">
          <string>Float merge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mergeBtn">
         <property name="text">
          <string>Merge + Rebin</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Merge all with precision:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>precisionEdit</tabstop>
  <tabstop>rebinBtn</tabstop>
  <tabstop>addConstEdit</tabstop>
  <tabstop>addBtn</tabstop>
  <tabstop>scaleConstEdit</tabstop>
  <tabstop>mulBtn</tabstop>
  <tabstop>shiftConstEdit</tabstop>
  <tabstop>shiftBtn</tabstop>
  <tabstop>monscaleEdit</tabstop>
  <tabstop>monscaleBtn</tabstop>
  <tabstop>mergeEdit</tabstop>
  <tabstop>onemodelBox</tabstop>
  <tabstop>onemodelBtn</tabstop>
  <tabstop>paramsetBtn</tabstop>
  <tabstop>globalfitBtn</tabstop>
  <tabstop>mappingBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/gui.qrc"/>
 </resources>
 <connections/>
</ui>
